tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -324,6 +326,10 @@
     import {fetchRuns} from "../tf-backend/runsStore.js";
     import * as paginatedViewStore from "../tf-paginated-view/paginatedViewStore.js";
 
+    if (Object.keys(dashboardRegistry).length === 0) {
```

Comment:
nit: Since we imported lodash, we could optionally use `_.isEmpty(dashboardRegistry)`

Reply:
Done